src/invoke.rs

```diff
@@ -12,6 +18,8 @@ use crate::strval::{self, StrValError};
 pub struct Invoke {
     #[clap(long, parse(from_os_str))]
     file: std::path::PathBuf,
+    #[clap(long, parse(from_os_str))]
+    snapshot_file: std::path::PathBuf,
```

Comment:
One thing left, could we set a default value for this field? Ideally it is something as a default, like `ledger.xdr`.

Reply:
Good idea. Done.